lib/cover_my_meds/api_request.rb

```diff
@@ -25,8 +25,14 @@ def request(http_method, host, path, params={}, auth_type = :basic, &block)
       rest_resource = RestClient::Resource.new(uri.to_s, tail)
 
       response = call_api http_method, rest_resource, &block
+      parse_response response
```

Comment:
Could make this `parse_response response if response.body.present?` and remove line 28 altogether.  Generally ifs in the first line of a method call are a code smell, I believe.